src/templates/homepage.html

```diff
@@ -8,6 +9,29 @@
   <p class="desktop-promo-separator-text">{{ _('More to Explore') }}</p>
 {% endif %}
 
+{% defer (url=anonApi('search')|urlparams(limit=11, doc_type='website'), pluck='objects', id='feed-websites') %}
+  {% if this %}
+    <div class="feed feed-items feed-home c" data-columns>
+    {% if this.length >= 3 %}
+      {{ website_tabs(this|sliceArray(0, 3)) }}
+    {% endif %}
+    {% if this.length >= 6 %}
+      {{ website_carousel(this|sliceArray(3, 12)) }}
+    {% endif %}
+    </div>
+  {% endif %}
+{% placeholder %}
+  <p class="spinner padded alt"></p>
+{% except %}
+  <p class="no-results">
+    {% if error == 400 or error == 404 %}
```

Comment:
don't think we need these errors messages. if this happens, something is wrong and we'd need to deploy something asap